.circleci/config.yml

```diff
@@ -41,8 +51,20 @@ workflows:
       - "images":
           context: falco
           filters:
+            tags:
+              only: /v[0-9]+(\.[0-9]+)*(-.*)*/
```

Comment:
Do we want tags to be `v` prefixed?

Reply:
Git tags need a `v` prefix to be recognized as a Go module version
See: https://github.com/golang/go/issues/24007#issuecomment-377603850

As far as I know, go mod will complain otherwise if someone tries to import driverkit as a package. Not sure if this behavior has changed recently, anyway I would keep the `v` for wide compatibility.